STC-DCP 2nd phase/content/sass/shared/_fonts.scss

```diff
@@ -0,0 +1,8 @@
+@font-face {
```

Comment:
you can add another 2` font-faces` with the missing 2 font links **(Medium , Bold)** so you can use the bold and medium style of the font itself.
Example:
```
@font-face {
    font-family: "STC forward ";
  src: url(../fonts/STC\ Forward\ Bold.woff) format("woff");
  font-weight: 900;
}
```